src/Tools/BuildActionTelemetryTable/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using TelemetryInfo = System.Tuple<string, string, string>;
+
+namespace BuildActionTelemetryTable
+{
+    class Program
+    {
+        private static readonly string s_executingPath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+
+        private static readonly string[] s_excludeList = new[]
+        {
+            "Microsoft.CodeAnalysis.AnalyzerUtilities.dll"
```

Comment:
what is the reason behind excluding this?

Reply:
I don't recall but it doesn't seem necessary anymore. Will remove it.